Data/Sys/GameSettings/GZ2.ini

```diff
@@ -31,3 +31,74 @@ EnableGPUTextureDecoding = False
 
 [Video_Enhancements]
 ArbitraryMipmapDetection = True
+
+[Gecko]
+$Bloom x4 IR
```

Comment:
This shouldn't be in a three-character INI unless it has been tested to be region free.